apps/web/src/features/resources/components/DropdownMenu/dropdown-menu.tsx

```diff
@@ -0,0 +1,106 @@
+import { useEffect, useRef, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { MdKeyboardArrowDown } from 'react-icons/md';
+
+import { cn } from '@sovereign-university/ui';
+
+import { CategoryIcon } from '#src/components/CategoryIcon/index.js';
+import { capitalizeFirstWord } from '#src/utils/string.js';
+
+import { RESOURCES_CATEGORIES } from '../../utils.tsx';
+
+import { ResourcesDropdownItem } from './dropdown-item.tsx';
+
+interface ResourcesDropdownProps {
+  resourceCategoryActive?: string;
+}
+
+export const ResourcesDropdown = ({
```

Comment:
I do not like having the name of the component not matching the name of the file.